pipeline/brew/brew.go

```diff
@@ -37,10 +37,22 @@ const formula = `class {{ .Name }} < Formula
   {{- end }}
   {{- end }}
 
+  {{- if .Install }}
+
+  def install
+    {{- range $index, $element := .Install }}
+    {{ . -}}
+		{{- end }}
+  end
+
+	{{- else }}
```

Comment:
wouldn't it be simpler to set the default `bin.install "binary"` when the config doesn't set anything?

If so, you could just remove the entire `if` block, because the `range` would deal with everything, no?